documentation/src/docs/python.md

```diff
@@ -0,0 +1,130 @@
+# Python API for the connector
+
+This Pravega connector of Python API provides a data source and data sink for Flink streaming jobs.
+
+Your Flink streaming jobs could use Pravega as their storage with these [Python API Wrappers](https://github.com/pravega/flink-connectors/tree/master/src/main/python).
+
+**DISCLAIMER: This python wrapper is an IMPLEMENTATION REFERENCE and is not meant for out-of-box usage.**
+
+[TOC]
+
+## How to use
+
+Together with python wrapper files, you could submit your job with main compute code like this:
+
+```bash
+flink run --python ./application.py --pyFiles ./pravega_config.py --pyFiles ./pravega_writer.py --pyFiles ./pravega_reader.py --jarfile /path/to/pravega-connectors-flink.jar
```

Comment:
Explain that users need to attach both the connector jar with `--jarfile` option and the python wrappers with `--pyFiles` option.
`--pyfiles` option supports both comma separated list and directory, no need to repeat this. Use directory like `<connector-repo>/src/main/python/` would be sufficient here.